packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -1280,9 +1296,12 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 			this.mc.logger.sendTelemetryEvent({
 				eventName: "GC_DeletingLoadedDataStore",
 				...tagCodeArtifacts({
-					id: dataStoreId,
+					id: `/${dataStoreId}`,
```

Comment:
Add a comment? This is to make it consistent with GC node path format right?

Reply:
yes. To make it consistent with other places id is logged. I will add a comment.